### ISSUE TITLE
SpanBERT: Improving Pre-training by Representing and Predicting Spans

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#110]
# TL;DR
While BERT hides one token (like a word) with a mask and fills it in, Span-BERT hides consecutive tokens (like an idiom) and fills them in. In addition, it does not perform Next Sentence Prediction and only makes the user input a single sentence. It outperforms BERT in many tasks.
![£(foot ball) = LMLM(football) +  LsBo (football)](https://user-images.githubusercontent.com/24524018/93177614-0cb6ef00-f76e-11ea-815d-ddfa62eb0abe.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/1907.10529

# Submission Dates(yyyy/mm/dd)
2019/07/24

# Authors and institutions
Mandar Joshi, Danqi Chen, Yinhan Liu, Daniel S. Weld, Luke Zettlemoyer, Omer Levy
- Allen School of Computer Science & Engineering, University of Washington, Seattle, WA
- Computer Science Department, Princeton University, Princeton, NJ
- Allen Institute of Artificial Intelligence, Seattle
- Facebook AI Research,

# Methods

# Results

# Comments
